src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
 body {
+  font-family: "Roboto", sans-serif;
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &__img {
+    display: block;
+    margin: auto;
+    width: 160px;
+    height: 134px;
+    padding: 32px 16px 40px;
+  }
+  &__info {
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+  }
+  &__title {
+    margin: 0;
+    padding: 0 16px 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+  &-code {
+    margin: 0;
+    padding: 0 16px 16px;
+    font-weight: 400;
```

Comment:
400 it's a default font-weight, so it's redundant here 
please check other places